### PR TITLE
Add release notes for 8.0 beta

### PR DESCRIPTION
**Pull request by eternoendless**
| Questions     | Answers
| ------------- | -------------------------------------------------------
| Description?  | Add release notes for beta
| Fixed ticket? | ~

To do:

- [x] Add examples of issues fixed
- [x] Add list of known issues
- [x] Review contributors and number of PRs

### CODE REVIEW

File: news/_posts/releases/2022-08-08-prestashop-8-0-beta-release.md [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+## Download
+
+You can download PrestaShop 8.0 beta here:
+
+[Download PrestaShop 8.0 beta now!](https://github.com/PrestaShop/PrestaShop/releases/tag/8.0.0-beta){: .btn .btn-lg .btn-success}
+{: .text-center}
+
+## Known issues
+
+ADD IMPORTANT BUGS
```
Comment (eternoendless) [inline news/_posts/releases/2022-08-08-prestashop-8-0-beta-release.md]: The list of most important known bugs goes here

File: news/_posts/releases/2022-08-08-prestashop-8-0-beta-release.md [hunk 2]
```diff
@@ -0,0 +1,105 @@
...
+
+PrestaShop 8 includes several additional built-in modules: Top-sellers block, Brand list, Cash on delivery, Products in the same category, Mail alerts, Google analytics, New Products, Specials block, Supplier list, Viewed products block, and Distribution API client. 
+
+During the installation process, you can now choose which modules you want to install.
+
+## Download
+
+You can download PrestaShop 8.0 beta here:
+
+[Download PrestaShop 8.0 beta now!](https://github.com/PrestaShop/PrestaShop/releases/tag/8.0.0-beta){: .btn .btn-lg .btn-success}
```
Comment (eternoendless) [inline news/_posts/releases/2022-08-08-prestashop-8-0-beta-release.md:88]: Check that the link is correct after the tag has been created

File: news/_posts/releases/2022-08-08-prestashop-8-0-beta-release.md (hunk 1 above)
Comment (eternoendless) [reply news/_posts/releases/2022-08-08-prestashop-8-0-beta-release.md]: @MatShir @Robin-Fischer-PS could you please help out here?

File: news/_posts/releases/2022-08-08-prestashop-8-0-beta-release.md [hunk 3]
```diff
@@ -0,0 +1,111 @@
...
+
+Also note that it is not possible to upgrade a shop from a beta to the final version using the upgrade module.
+
+## Your feedback is important!
+
+The public beta period is an extremely important phase to prepare for the upcoming release. During this period, we encourage everyone in the community to download this version and test how it behaves in their unique environment.
+
+Here are some examples of how you can help:
+
+* If you have access to a real shop, set up a copy of it in a separate environment and update it to PrestaShop 8 using the [upgrade module](https://github.com/PrestaShop/autoupgrade/releases). See if you notice any issues on the shop compared to your previous version.
```
Comment (MatShir) [inline news/_posts/releases/2022-08-08-prestashop-8-0-beta-release.md]: ps_metrics, ps_facebook, Prestashop Checkout, Google marketing, welcome, mbo

Module list to uninstall before upgrading

**Comment by ghost** [comment]
Just before merge can you see
https://prestashop.slack.com/archives/CTKG1NKAP/p1659951845186859
For no broken link

File: news/_posts/releases/2022-08-08-prestashop-8-0-beta-release.md [hunk 4]
```diff
@@ -0,0 +1,121 @@
...
...brahima Sow, idnovate, Igor Stępień, Jean-François Viguier, Jérôme Wohlschlegel, Jevgenij Visockij, Jonathan Danse, Jonathan Lelievre, Jory Hogeveen, Julie Varisellaz, Julius Žukauskas, Karlis Suvi, Kévin Gleizes, Khouloud Belguith, Krystian Podemski, Kyary, Lionel Massin, Lorenz Meyer, Louis Authie, Luc Vigato, Mahmoud Parvazi, Marco Ingraiti, Marco Salvatore, Marcos, Marek Jedrzejewski, Marion François, Marius Gudauskis, Marwa Chelly, Mathieu Ferment, Matthias Raigne, Matthieu Rolland, Michael Voříšek, Mickaël Fernandez, Moncef Essid, mrAKAR, Nesrine Abdmouleh, Nicolas Sorosac, NoZTurn, okom3pom, Olivier Le Corre, Pablo Borowicz, Pascal Cescon, Pedro Nucci, Peter Schaeffer, Philip Iezzi, Pierre Rambaud, Pierre-Yves Christmann, PrestaAlba, prestaforum, Prestaplugins, Prestaworks, Priyank Desai, Psyray, PululuK, Ricardo González Garza, Robin Fischer, Romain ‘Maz’ BILLOIR, Romain Couderc, Sam, Sauli Maijala, Sébastien Alfaiate, Sergio Quiñonez, Seth Vargo, Sharak, Simone, Simone Gennari, Tatu Wikman, Thomas Baccelli, Thomas Leguillant, Tom Combet, Tuni-Soft, Valentin Szczupak, venditdevs, Vincent Garcia, Viorel Preoteasa, Vladimir, Web Helpers Pau, Yneet, yo7hli, and Yohann Tilotti.
```
Comment (kpodemski) [inline news/_posts/releases/2022-08-08-prestashop-8-0-beta-release.md]:
```suggestion
A big thank you to the **113 people** who contributed **more than 1000 Pull Requests** included in this version: Adib Aroui, Alexandre Bru, Amaury Hanser, Amazzing, Amit Kumar Tiwari, Andromaque, Arnaud104, Ashley Butcher, Axome, Beattie282, Bertrand Merlier, Boubker Bribri, Brais Pato, Bruno Desprez, Camille V, Christophe Rouille, Clotaire Renaud, Constantin Boulanger, Damien Tupinier, Daniel Hlavacek, David Diverres, David Gonzalez, David Oury, Dominik Ulrich, e-gaulue, Fabien Papet, Franck Lefèvre, gennaris, Georges Cubas, Henri Davigne, Hervé Hennes, Hibatallah Aouadni, Hugues Gobet, Ibrahima Sow, idnovate, Igor Stępień, Jean-François Viguier, Jérôme Wohlschlegel, Jevgenij Visockij, Jonathan Danse, Jonathan Lelievre, Jory Hogeveen, Julie Varisellaz, Julius Žukauskas, Karlis Suvi, Kévin Gleizes, Khouloud Belguith, Krystian Podemski, Kyary, Lionel Massin, Lorenz Meyer, Louis Authie, Luc Vigato, Mahmoud Parvazi, Marco Ingraiti, Marco Salvatore, Marcos, Marek Jedrzejewski, Marion François, Marius Gudauskis, Marwa Chelly, Mathieu Ferment, Matthias Raigne, Matthieu Rolland, Michael Voříšek, Mickaël Fernandez, Moncef Essid, mrAKAR, Nesrine Abdmouleh, Nicolas Sorosac, NoZTurn, Olivier Le Corre, Pablo Borowicz, Pascal Cescon, Pedro Nucci, Peter Schaeffer, Philip Iezzi, Pierre Rambaud, Pierre-Yves Christmann, PrestaAlba, prestaforum, Prestaplugins, Prestaworks, Priyank Desai, Psyray, PululuK, Ricardo González Garza, Robin Fischer, Romain ‘Maz’ BILLOIR, Romain Couderc, Sam Berry, Sauli Maijala, Sébastien Alfaiate, Sergio Quiñonez, Seth Vargo, Sharak, Simone, Simone Gennari, Tatu Wikman, Thomas Baccelli, Thomas Leguillant, Tom Combet, Tuni-Soft, Thomas Roux, Valentin Szczupak, venditdevs, Vincent Garcia, Viorel Preoteasa, Vladimir, Web Helpers Pau, Yneet, yo7hli, and Yohann Tilotti.
```

Comment (kpodemski) [reply news/_posts/releases/2022-08-08-prestashop-8-0-beta-release.md]: uhh, the diff is broken, I've changed `Sam` to `Sam Berry` and `okom3pom` to `Thomas Roux`

Comment (ghost) [reply news/_posts/releases/2022-08-08-prestashop-8-0-beta-release.md]: Okom3pom = Thomas Roux :+1:

**Comment by PrestaEdit** [comment]
Let met say that you are doing the worst move with that... 😢

**Comment by eternoendless** [comment]
Care to develop your thoughts @PrestaEdit?

**Comment by PrestaEdit** [comment]
Making a beta version of a major software, beginning of August and close the door of feebacks beginning of September is not a good move to do. I hope to be not in the good (and I already know that it's not the truth) but there is still some testers on vacations and they will have not such time to try and give feedback on this.

Hope for your that you will have some (good) feedbacks on the right time.

**Comment by eternoendless** [comment]
Yes, we know the timing is less than ideal because many people are on vacation during August. 

Unfortunately, we are already very late compared to our original schedule, and we rather risk low engagement during the beta period rather than delay the beta to September, and end up being even later for the final version.

**Comment by PrestaEdit** [comment]
It's still be possible to delay. There is no need to launch a specific version on specific date, isn't it ? Or the needs its another.
PrestaShop 1.7 was released because of this reason : still not finish, as you know better than me.

I don't understand this range of dates, to be honest.

**Pull request merged**